tests/sentry/search/test_utils.py

```diff
@@ -514,3 +515,17 @@ def test(self):
             environment = self.create_environment()
             result = get_latest_release([self.project], [environment])
             assert result == new.version
+
+
+class ConvertUserTagTest(TestCase):
```

Comment:
Thanks for adding direct tests, I guess I tested this via serializer when I added it